api/buckets/service.go

```diff
@@ -1166,24 +1203,71 @@ func getLink(lnks []*ipld.Link, name string) *ipld.Link {
 // updateOrAddPin moves the pin at from to to.
 // If from is nil, a new pin as placed at to.
 func (s *Service) updateOrAddPin(ctx context.Context, from, to path.Path) error {
+	key, isUser := ownerFromContext(ctx)
+
+	var futureSize int64
+	if !isUser {
```

Comment:
So, it seems that if the context is from a User, then there's no `Account` in the collection, thus we can't track the size of the total buckets. Is that correct @sanderpick ?  I realized this from some tests failing.
Sounds a bit unfortunate, but maybe we can solve this case in the future. Not completely sure what would be the best solution.